test/WebJobs.Script.Tests.Integration/Storage/StorageEndToEndTests.cs

```diff
@@ -48,7 +50,9 @@ protected override ExtensionPackageReference[] GetExtensionsToInstall()
                     new ExtensionPackageReference
                     {
                         Id = "Microsoft.Azure.WebJobs.Extensions.Storage",
-                        Version = "3.0.10"
+                        Version = "4.0.0"
+                        //Version = "4.0.0-dev637091944077950638"
+                        //Version = "3.0.10"
```

Comment:
nit -- remove comments.

Reply:
It looks like this is actually failing to install the storage extension in CI. So I will need to fiddle with this some more before cleaning up.